### ISSUE TITLE
Nodebox installs itself without querying the User for the target directory

### ISSUE THREAD

**Issue by rebb** [nodebox/nodebox#449]
Is this intended behaviour ?
I don't think you can expect every user to want to install to C:\Program Files - in my case i certainly don't.

This just feels like the program is trying to make decisions for the user, trying to know what is "best" for him/her.
But ultimately it takes away control.

Thankfully 99.9% of software don't do this, and there's a reason for it.

**Comment by fdb** [comment]
Thanks for reporting this. We recently switched installers and the installer automatically selects the destination directory, as a convenience for the user.

I guess there is an option to turn directory selection on, I'll see if I can find it.

**Comment by PabloCaracena** [comment]
Hi, thanks for the software!
Just want to say my installation experience was not good. It didn't show any finish notification, and the windows installer was still running.

I tried to install for the first time, and the message of "Wait while windows configure.." appeared. When it finished it just disappeared, and **the Task Manager was still showing the windows installer**, but no CPU usage, so I finished it after a while. After trying this 2 more times I came here and after seeing this post, I checked it and it is installed! Hahahaha wtf

Thanks for the free software anyways!! I was looking for something like this

**Comment by fdb** [comment]
Yeah we have very little control over the installer at the moment...
